### ISSUE TITLE
R Markdown pandoc.exe failure when running on networked drive with renv

### ISSUE THREAD

**Issue by joshpk** [rstudio/renv#68]
Hi,
I'm testing implementing the renv package for managing packages and I have an issue when my R project is on a networked drive. Using renv, if I try to knit an R markdown document locally on my C drive it works as expected. However, if I knit the same document when the project is located on a networked drive I get the following error:

> pandoc.exe: Could not fetch \\joshua\projects\Temp\test_project\renv\library\x86_64-w64-mingw32\3.5\rmarkdown\rmd\h\jquery\jquery.min.js
FailedConnectionException2 "joshua" 80 False connect: failed (Connection refused (WSAECONNREFUSED))
Error: pandoc document conversion failed with error 67
Execution halted

Code: 

```
---
title: "test_renv"
author: "joshpk"
date: "April 23, 2019"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

Any ideas on how to resolve this? Thanks.

**Comment by kevinushey** [comment]
Are you sure this is related to `renv`? This sounds like a more general problem with `rmarkdown` / `pandoc` + networked drives. https://github.com/ramnathv/rcstatebin/issues/6 seems related.

What version of Pandoc are you using?

**Comment by joshpk** [comment]
You are right, sorry - it was strange because I did not get an issue with this before I setup renv. Anyway, I updated R Studio to the latest version which is bundled with pandoc v2.6 and then mapped the network drive to a letter, which seems to have fixed the issue.
